proposals/0111-linear-types.rst

```diff
@@ -1253,6 +1387,101 @@ interpret GADT-syntax type declaration (both ``data`` and ``newtype``)
 in code without ``-XLinearTypes`` to be *linear*, despite the
 ostensible use of an unrestricted arrow.
 
+Let bindings and polymorphism
+-----------------------------
+
```

Comment:
Refer back to the clause in the spec that says "linear bidnigns with multipiclity annotations are monomorphic"

And fwd ref to this section from the spec